src/screens/BookDetail/styles.module.scss

```
@@ -0,0 +1,73 @@
+@import 'scss/components';
+
+.container {
+  box-shadow: 0 4px 10px 0 rgba($black, 0.5);
+  padding: 1.5rem;
+  margin-top: 5rem;
+  min-width: 900px;
+}
+
+.label {
+  color: $black;
+  font-weight: bold;
+}
+
+.titleContainer {
```

the compound classes must be written in kebab case
```suggestion
.title-container {
```